### PR TITLE
feat(phone-number): implemented phone number support

### PR DESCRIPTION
**Pull request by wim07101993**
## Connection with issue(s)

Resolve issue #33 

## Testing and Review Notes

All previous unit tests should pass. Added a new test for generating phone numbers.

### CODE REVIEW

File: test/specs/phone_number.dart [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import 'package:faker/faker.dart';
+import 'package:test/test.dart';
+
+void main() {
+  var faker = Faker();
+
+  group('person', () {
```
Comment (drager) [inline test/specs/phone_number.dart]: Should be phoneNumber

File: test/specs/phone_number.dart [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    test('should be able to generate phone number', () {
+      for (var i = 0; i < 20; i++) {
+        expect(
+          faker.person.name(),
```
Comment (drager) [inline test/specs/phone_number.dart]: Shouldn't be testing `faker.person.name()`?

**Review by drager — CHANGES_REQUESTED** [review]
Thank you! Just some thoughts below

File: test/specs/phone_number.dart (hunk 2 above)
Comment (wim07101993) [reply test/specs/phone_number.dart]: ... indeed

**Review by drager — APPROVED** [review]

**Comment by firatcetiner** [comment]
PhoneNumber() cannot be used in `1.3.0`. @drager When do you plan to publish a new version that includes this PR? In the meantime, I will have to fetch the package from `git` instead of `pub`.

**Comment by drager** [comment]
@firatcetiner Will publish 1.4.0 today! Sorry for the delay!

**Comment by drager** [comment]
@firatcetiner Please try out [2.0.0-rc-1](https://pub.dev/packages/faker/versions/2.0.0-rc.1)

**Comment by firatcetiner** [comment]
@drager Thanks. I guess I will stick with the old version because I can not switch to the Flutter beta since `2.0.0-rc-1` requires Dart SDK version to be `>=2.12.0` and the current Flutter stable channel has `2.10.5`.

**Pull request merged**